test/simpleGreetingTest.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var expect = require('chai').expect;
+var test = require('../lib/simpleGreeting.js');
+
+describe('greet function', function(name){
+	it('should return hello name', function(){
```

Comment:
`name` is undefined. use a specific value:
`expect(test.greet('beep')).to.eql('hello beep');`